apollo-router/src/plugins/telemetry/proto/reports.proto

```diff
@@ -498,6 +498,15 @@ message ContextualizedStats {
 
 }
 
+message QueryMetadata {
```

Comment:
Yep I was definitely planning to wait until the changes were deployed to usage-reporting.api.apollographql.com before making any of these changes. The process I'm proposing and following can be found [here](https://github.com/mdg-private/monorepo/pull/13468/files#diff-ae1706c4818260d41095a2f58c105647c1b1debaece2b318c36e14c9a7f5d8fd). I thought that once the updated proto was deployed to usage-reporting.api.apollographql.com, we still need to update the file in a router PR though? This is the message that comes up if you try to build/test after the change is deployed to usage-reporting.api.apollographql.com. 

```
---- plugins::telemetry::apollo_exporter::check_reports_proto_is_up_to_date stdout ----
thread 'plugins::telemetry::apollo_exporter::check_reports_proto_is_up_to_date' panicked at 'Protobuf file is out of date. Run this command to update it:

    curl -f https://usage-reporting.api.apollographql.com/proto/reports.proto > apollo-router/src/plugins/telemetry/proto/reports.proto

', apollo-router/src/plugins/telemetry/apollo_exporter.rs:394:5
stack backtrace
```

Reply:
That's correct. I was just observing that we need to make the "upstream" change first and then update in our repo. Otherwise changes would potentially be lost when we downloaded an "upstream" change.